charmcraft/utils.py

```diff
@@ -227,3 +229,17 @@ def confirm_with_user(prompt, default=False) -> bool:
         return False
     else:
         return default
+
+
+def format_timestamp(dt: datetime.datetime) -> str:
+    """Convert a datetime object (with or without timezone) to a string.
+
+    The format is
+
+        <DATE>T<TIME>Z
+
+    Always in UTC.
+    """
+    # convert to UTC no matter the timezone `dt` has
```

Comment:
Maybe explain somewhere where our datetimes come from and why we can ignore the timezone?

Reply:
Mm... we're not ignoring the timezone.

Reply:
There I improved the comment